assets/src/edit-story/components/form/advancedDropDown/dropDown.js

```diff
@@ -19,16 +19,14 @@
  */
 import { useState, useCallback, useRef } from 'react';
 import { useDebouncedCallback } from 'use-debounce';
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
-import { rgba } from 'polished';
-import { __ } from '@web-stories-wp/i18n';
 
 /**
  * Internal dependencies
  */
-import { ReactComponent as DropDownIcon } from '../../../icons/dropdown.svg';
 import Popup from '../../popup';
+import DropDownSelect from '../../../../design-system/components/dropDown/select';
```

Comment:
Not completely sure if this change is related to this visual bug, but when you select an item and long-press the option, the layout shifts (see the dropdown height, it's probably due to the focus outline) until you release the mouse and the dropdown is closed.

It looks odd:

![dropdown-focus-bug](https://user-images.githubusercontent.com/637548/111240904-7ce5e680-85d2-11eb-9ed2-084426a0a049.gif)

Reply:
There's no change to that part at all -- we still don't have the right dropdown component to use for this so currently, the PR is just switching out the button but not the list.

The same issue happens on `main`, too.